patches/components-history-core-browser-thumbnail_database.cc.patch

```diff
@@ -0,0 +1,31 @@
+diff --git a/components/history/core/browser/thumbnail_database.cc b/components/history/core/browser/thumbnail_database.cc
+index 19c7722a778b24fcc50e7ab9a4e5e368399bf52d..625f5be347adbbfeb207ba45880372435b1a4e80 100644
+--- a/components/history/core/browser/thumbnail_database.cc
++++ b/components/history/core/browser/thumbnail_database.cc
+@@ -519,6 +519,7 @@ FaviconBitmapID ThumbnailDatabase::AddFaviconBitmap(
+     base::Time time,
+     const gfx::Size& pixel_size) {
+   DCHECK(icon_id);
++  type = FaviconBitmapType::ON_DEMAND; // Make all icons ON_DEMAND.
+ 
+   sql::Statement statement(db_.GetCachedStatement(
+       SQL_FROM_HERE,
+@@ -573,8 +574,8 @@ bool ThumbnailDatabase::SetFaviconBitmap(
+   } else {
+     statement.BindNull(0);
+   }
+-  statement.BindInt64(1, time.ToDeltaSinceWindowsEpoch().InMicroseconds());
```

Comment:
I don't think this code is ever reached because of https://github.com/brave/brave-core/pull/6435/files#diff-9357d51839b49d0d87f163071368029cR28